nvtabular/loader/tensorflow.py

```diff
@@ -305,8 +305,8 @@ def _handle_tensors(self, cats, conts, labels):
         # would require output layers to match naming
         if len(self.label_names) > 1:
             labels = tf.split(labels, len(self.label_names), axis=1)
-        else:
-            labels = [labels]
+        # else:
+        #     labels = [labels]
```

Comment:
Lets just delete this:
```suggestion
```

Reply:
Yeah I have this just straight up removed in my mh branch, fine with that